packages/core/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@eslint/core",
+  "version": "0.0.1",
+  "description": "Runtime-agnostic core of ESLint",
+  "type": "module",
+  "types": "./src/types.ts",
+  "files": [
+    "src"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "tsc",
```

Comment:
I'm not sure what's the intent of the build step, because it only generates a `dist/esm/types.d.ts` file but we're not publishing that file.

Reply:
It's to match other packages, where you can run `npm run build` to validate the TypeScript.